nats.go

```diff
@@ -3268,17 +3273,53 @@ func badQueue(qname string) bool {
 }
 
 // subscribe is the internal subscribe function that indicates interest in a subject.
-func (nc *Conn) subscribe(subj, queue string, cb MsgHandler, ch chan *Msg, isSync bool) (*Subscription, error) {
+func (nc *Conn) subscribe(subj, queue string, cb MsgHandler, ch chan *Msg, isSync bool, opts ...ConsumerOption) (*Subscription, error) {
```

Comment:
Here's an example - if we had the subscribe option, the sig would be stable.

Reply:
Yeah, agree